parachain/pallets/omni-account/src/lib.rs

```diff
@@ -387,10 +439,13 @@ pub mod pallet {
 
 		#[pallet::call_index(6)]
 		#[pallet::weight((195_000_000, DispatchClass::Normal))]
-		pub fn request_intent(origin: OriginFor<T>, intent: Intent) -> DispatchResult {
+		pub fn request_intent(
+			origin: OriginFor<T>,
+			intent_id: IntentId,
+			intent: Intent,
+		) -> DispatchResult {
 			let who = T::OmniAccountOrigin::ensure_origin(origin)?;
-			Self::deposit_event(Event::IntentRequested { who, intent });
-			Ok(())
+			Self::do_accept_intent(who, intent_id, intent)
```

Comment:
Can cross chain swap be accepted without balance checks?

Reply:
We could add back the `IntentRequested` event and only call accept_intent after sanity check (or just get rid of this call)

Reply:
The initial idea was to make it possible to submit intent directly onto chain (for non-pumpx cases), but it could cause chaos in this case.

I'll just remove this ext for now